### ISSUE TITLE
Symbols aren't appearing correctly in the ios app for DLF ND

### ISSUE THREAD

**Issue by rm51** [udacity/dlnd-issue-reports#213]
In Gradient Descent: The Code the formulas aren't appearing correctly on ios. Also in other places in that lesson there is the same problem with formulas

**Comment by rm51** [comment]
![gradient_descent](https://cloud.githubusercontent.com/assets/7563076/24327160/c57cee1a-1197-11e7-8821-60c8466400e0.PNG)

**Comment by rm51** [comment]
![implementing_gradient_descent](https://cloud.githubusercontent.com/assets/7563076/24327163/df05e4ae-1197-11e7-8eb0-fe8bd1976c49.PNG)

**Comment by rm51** [comment]
![backpropagation](https://cloud.githubusercontent.com/assets/7563076/24327166/ff3386fa-1197-11e7-8361-704331ad9bf2.PNG)

**Comment by clapollo** [comment]
Ah, it looks like whatever they're using to render things in iOS isn't handling the latext commands we use. 

I'm assigning this to you, @oritavital, so you can figure out the right way to address this with the mobile team.

**Comment by oritavital** [comment]
Thanks for the feedback, I reported that issue to our mobile team

Orit
